minutes/sync-meeting/2023-09-07.md

```diff
@@ -0,0 +1,72 @@
+# 2023-09-07
+
+## Agenda
```

Comment:
I think there were some private discussions here. Can we at least mark that these were had. Perhaps in the future when the details are no longer sensitive, we can add a note here of what was discussed.

Reply:
Sure. I had noted that below, but I also copied it up here, too.